src/components/gallery/PinkFlower.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import Image from "next/image";
+import pinkFlower from "@/public/gallery/pinkFlower.webp";
+
+const PinkFlower = () => {
+  return (
+    <div>
+      <Image
+        src={pinkFlower}
+        alt="pinkFlower"
+        className="absolute bottom-0 left-0 translate-x-[-12.5%] translate-y-[12.5%] scale-75"
```

Comment:
by adding scale, the width/height are being messed up which is why you need to translate x and y, all you need to do is modify the width on top of the absolute positioning
```suggestion
        className="absolute bottom-0 left-0 w-2/12"
```